testing/run_tests.py

```diff
@@ -45,7 +45,7 @@
 
 LOG_FILE = os.path.join(OUT_DIR, 'run_tests.log')
 logger = logging.getLogger(__name__)
-console_logger_handler = logging.StreamHandler()
+console_logger_handler = logging.StreamHandler(sys_stdout)
```

Comment:
Consider adding a comment here explaining why this is needed.

Reply:
done